Updating data analysis topic based on latest changes (#1719)
IgniteUI/igniteui-docfx: en/components/general/framework-and-features/data-analysis.md

_keywords: data analysis, ignite ui for angular, infragistics
---

## Data Analysis

>  Data analysis is the process of examining, transforming, and arranging data in a specific way to generate useful information from it. Data analysis allows for the evaluation of data through analytical and logical reasoning to lead to some sort of outcome or conclusion in some context.

Reviewer: ```suggestion
Data analysis is the process of examining, transforming, and arranging data in a specific way to generate useful information based on it. It also allows for reaching certain outcomes and conclusions through analytical and logical reasoning.
```

>[!NOTE]
>This functionality will be introduced in **Ignite UI for Angular** as external package in order to ease the configuration and limit the required code at minimum

### Data Analysis Button
The data analysis button is the outlet to visualize your selected data in various ways:

![](../../../images/general/data-analysis-button.png)

> This way every range selection you perform in the grid is gaining importance and can be easily analyzed with just a button click.

Reviewer: ```suggestion
This way every range selection performed in the grid can be easily analyzed in a single click.
```

> The button renders on every range selection, its position is at the **bottom-right most fully visible cell** of the selection and hides when the range selection is no more active. Horizontal and vertical scrolling reposition the button so that it can always render at its designated position.

Reviewer: ```suggestion
The button is rendered on every range selection at the **bottom-right** of the selection and hides when the selection is inactive. Horizontal and vertical scrolling reposition the button so that it is always rendered at its designated position.
```

### Chart Integration

> This section introduces Grid's integration with charting functionality, which allows the end user to visualize a chart, based on Grid's selected data and choose different chart types if needed.

Reviewer: no comma needed I think.

The chart will be shown by selecting a range of cells and by clicking on the show analysis button.

>[!NOTE]

> >The chart creation option is available when there are numeric values in the selected data, otherwise it will be disabled.

Reviewer: ```suggestion
>The chart creation option is only available when there are numeric values in the selected data.
```

- ScatterPoint, ScatterBubble, ScatterLine. In order to show meaningful Scatter Bubble Chart we disable the preview when the data is not in valid format.
- [Pie Chart](../../pie-chart.md)

### Conditional Cell Formatting 
If you have a Grid with thousands of rows of data it would be very difficult to see patterns and trends just from examining the raw information. Similar to charts and sparklines, `Conditional formatting` provides another way to visualize data and make it easier to understand. 

> Understanding conditional formatting - it allows you to automatically apply formatting such as colors and data bars to cells based on `their value` in the range selection. The [sample below](#demo) demonstrates how you can configure the Grid to apply `Conditional Formatting`. It depends on the `Conditional formatting selection type` what condition `rules` will be shown.  Below you will find the predefined styles (presets) that you can use in order to quickly apply conditional formatting to your data. The formatting of a range clears when performing formatting on different range or through the clear button. The clear button activates when there is an applied formatting and removes the formatting from the whole range.  

Reviewer: it allows for applying formatting such as

Reviewer: The formatting of a range gets cleared when performing formatting on different range or through the clear button.

Reviewer: The clear button is only active when there is an applied formatting.
